compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text/selection/TextSelectionColors.kt

```diff
@@ -67,13 +67,5 @@ class TextSelectionColors(
  */
 val LocalTextSelectionColors = compositionLocalOf { DefaultTextSelectionColors }
 
-/**
- * Default color used is the blue from the Compose logo, b/172679845 for context
- */
-private val DefaultSelectionColor = Color(0xFF4286F4)
-
 @Stable
-private val DefaultTextSelectionColors = TextSelectionColors(
-    handleColor = DefaultSelectionColor,
-    backgroundColor = DefaultSelectionColor.copy(alpha = 0.4f)
-)
+internal expect val DefaultTextSelectionColors: TextSelectionColors
```

Comment:
By the way, we have a similar situation with TextField's cursor - we change its width for desktop. Maybe in the future we need to come up with some configurable approach, because selection color and the cursor width are design things. It is still a part of platform design, not of a widget catalog design (aka Material) though. Maybe we can introduce something like this:
```
PlatformTheme(
  text = TextTheme(
      cursorWidth = 1.dp,
      selectionColor = Color.Blue.copy(alpha = 0.04),
  ),
  contextMenuRepresentation = @Composable {

  },
  shadow = ShadowTheme(direction = ...)
) {

}
```
not sure.

It is just a note, the current code is perfect :)